### PR TITLE
[update] news rules + v1

### PR DESCRIPTION
**Pull request by yowainwright**
> Version bump + new rules

> New rules

---

<details>

<summary>Pull Request Todo</summary>

- [ ] Solicit reviews from domain experts or others outside this PR's squad

</details>

<details>

<summary>Post-Deploy Todo</summary>

- [ ] Monitor exceptions in Sentry – https://goo.gl/gB1t9Z
- [ ] Monitor key metrics like pixel fires and asset sizes – https://goo.gl/BkVpRg
- [ ] Spot-check production – http://dollarshaveclub.com

_In the event an outage related to this PR occurs, rollback immediately, then debug: https://jenkins-deploy.aws.shave.io/job/Production/job/rollback/_

</details>

<details>

<summary>Code Review Guidelines</summary>

- Ensure readability and good organization
- Ensure classes, functions, components, and variables are well-named
- Ensure the diff is small
- Ensure this PR does only one thing
- Ensure the code is DRY (more than twice repeated)
- Ensure tricky code is removed or explained with a comment
- Ensure `TODO` comments and `WHY` comments are added where applicable

</details>

### CODE REVIEW

File: react.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
-'use strict'
+"use strict";
 
 module.exports = {
-  extends: [
-    require.resolve('tslint-react'),
-    require.resolve('./'),
-  ]
-}
+  extends: [require.resolve("tslint-react"), require.resolve("./")],
```
Comment (briangonzalez) [inline react.js:4]: @yowainwright noice.

@kylecorbelli just added a few more to face-manager as well, fyi.

**Review by jonathanong — APPROVED** [review]

**Comment by jonathanong** [comment]
i think u need to rerun `eslint --fix` with latest changes u made

**Comment by yowainwright** [comment]
@jonathanong is your comment for `face-web`? 😜

**Comment by jonathanong** [comment]
@yowainwright no, i see `"`s being used

**Comment by yowainwright** [comment]
@jonathanong so, shall I add eslint as a dev dep for tslint?

**Comment by jonathanong** [comment]
oh... i thought we switched quotes because `eslint --fix` changed it. yeah, we should, but that can be a separate PR

**Pull request merged**